README.md

```diff
@@ -7,31 +7,45 @@
 [![Crates.io](https://img.shields.io/crates/v/tplay)](https://crates.io/crates/tplay)
 [![Crates.io](https://img.shields.io/crates/d/tplay)](https://crates.io/crates/tplay)
 [![License](https://img.shields.io/badge/license-MIT-blue)](LICENSE)
+[![Aur](https://img.shields.io/aur/version/tplay-git)](https://aur.archlinux.org/packages/tplay-git)
+
+<a href=""
+    ><img
+      alt="Maintenance"
+      src="https://img.shields.io/maintenance/yes/2023?style=flat-square"
+    />
+  </a>
+
```

Comment:
remove this!

Reply:
Done that :+1: .

Reply:
Sure, but you must remove this code, too. 

Otherwise there is still a "Maintenance" badge which - aside from the fact that has a different visual style - I do not want.


![image](https://github.com/maxcurzi/tplay/assets/30084738/fc982c1d-94a9-4222-92f8-7311abd3963e)